gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/TestKotlinVersion.kt

```diff
@@ -0,0 +1,8 @@
+package org.jetbrains.compose.test
+
+@Suppress("EnumEntryName")
+enum class TestKotlinVersion(val versionString: String) {
+    // __KOTLIN_COMPOSE_VERSION__
+    Default("1.4.31"),
+    V1_5_20_dev_3226("1.5.20-dev-3226")
```

Comment:
Shalln’t some recent 1.5.0 work here as well?

Reply:
I was not able to find any dev builds with the relevant change https://github.com/JetBrains/kotlin/commit/8124a9546729b05dee7b8ea489fdd5b29e60ef00

I don't think it worth spending a lot of time figuring out the relevant build